js/components/viewControlsVr.js

```diff
@@ -18,15 +18,42 @@ var ViewControlsVr = React.createClass({
       e.cancelBubble = true; // IE
       this.props.controller.state.accessibilityControlsEnabled = true;
     }
+
     this.props.controller.moveVrToDirection(isRotated, direction);
   },
 
+  /**
+   * @method ViewControlsVr#_setIconSymbol
```

Comment:
Using the word 'set' would mean you should be able to set the value to something that I pass in.  This should be named _setupIconSymbol or _initIconSymbol

Reply:
fix